### ISSUE TITLE
[Bug]: Cannot train locomotion policies with RL

### ISSUE THREAD

**Issue by rook86** [Genesis-Embodied-AI/Genesis#1230]
### Bug Description

Hi, I am following the section "Training Locomotion Policies with RL" from the document[https://genesis-world.readthedocs.io/en/latest/user_guide/getting_started/locomotion.html](url). However, I am having the issue that I cannot train go2-walking policy properly like shown in the attached video. It seems like the loss is reducing. Did anyone be able to train it properly? Thank you.

https://github.com/user-attachments/assets/40209a2a-be6d-4009-bc8f-415d9bda7abf

### Steps to Reproduce

python examples/locomotion/go2_train.py
python examples/locomotion/go2_eval.py

### Expected Behavior

https://github.com/user-attachments/assets/164bb925-6a41-4e4c-a08d-6d58915fc605

### Screenshots/Videos

![Image](https://github.com/user-attachments/assets/c153707c-ac4f-4f7b-86ed-da2c7b8924ae)

### Relevant log output

```shell

```

### Environment

- OS: Ubuntu 22.04
- GPU/CPU A100

### Release version or Commit ID

main (v0.2.1)

**Comment by duburcqa** [comment]
Strange. You can try again with the most up-to-date main branch? Some bugs related to URDF loading has been fixed recently.

**Comment by rook86** [comment]
@duburcqa 
Thank you for the reply. I have updated to the newest version, but still I cannot train properly. I think the reward is different from what is shown in the document. Is it possible to check on your environment?

![Image](https://github.com/user-attachments/assets/4f6de6ae-5003-4428-884e-80c9f7709c58)

**Comment by rook86** [comment]
@duburcqa 
I used the urdf file from [https://github.com/Derek-TH-Wang/mini_cheetah_urdf/tree/master](url) and was able to train properly. Do you want me to make a pull request?

https://github.com/user-attachments/assets/28eee0d3-c72f-48dd-a822-458cee6fe504

**Comment by duburcqa** [comment]
OK thank you. No I think we would rather want to fix the original training script. Bugs have been introduced recently (by me!) and I think fixing them all should fix training at the same time.

**Comment by rook86** [comment]
@duburcqa 
I see. Thank you. I'm looking forward for the update.

**Comment by duburcqa** [comment]
Hopefully this time it is actually fixed ! If not, then I will look more closely at the RL training script in particular. Could you have a look ?

**Comment by tykim0507** [comment]
@duburcqa 
Hello, thank you for your hard work. I have used the latest version, but unfortunately encountered the same bug.
Below are the graphs logged with tensorboard, and it seems identical to @rook86 's result.
![Image](https://github.com/user-attachments/assets/981ea0ef-08a5-46b9-9709-07e69a1e4bad)
![Image](https://github.com/user-attachments/assets/1ca5af2a-543a-484c-a497-4e2ca61153bd)

Also the simulation result shown by running the evaluation outputs the same result.

**Comment by duburcqa** [comment]
I see thank you for your valuable feedback. We will investigate this issue.

**Comment by duburcqa** [comment]
@tykim0507  Is it working again now? I think it should be!
